micrometer-core/src/main/java/io/micrometer/core/VisitableMeter.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2017 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.core;
+
+/**
+ * Accepts a {@link MeterVisitor} implementation.
+ */
+public interface VisitableMeter {
+    void accept(MeterVisitor visitor);
```

Comment:
I'm OK putting this on `Meter` itself.

Reply:
I rather leave this as a separate interface as it just shows the capability to accept a visitor (something which is not really a meter's concern). But if you prefer to avoid the proliferation of classes I can for sure add it the the `Meter`.